docs/release-notes/spring-2022/README.md

```diff
@@ -11,53 +11,59 @@
   ],
 }
 ---
+<cdr-banner type="info" aria-live="polite"><template #icon-left><icon-information-fill inherit-color /></template><span style="font-weight:500">This is our final major release prior to Cedar's migration to Vue 3</span><template #message-body>Cedar's next release will migrate our component library to Vue 3. As a result, this will be the only Vue 2 release we will support post-migration.</template></cdr-banner>
```

Comment:
This is the final major release for Vue 2 before Cedar's migration to Vue 3. In our next release, we will complete the transition by migrating our component library to Vue 3. **Please note:** This will be the only Vue 2 release we are supporting post-migration.